### ISSUE TITLE
Feature request: Search/parse/regex in access logs

### ISSUE THREAD

**Issue by marcelomd** [openresty/test-nginx#29]
Hi,

It would be nice to be able to check the access logs for strings, regexes, etc, just like the error logs.

Thanks a lot for this module!

**Comment by agentzh** [comment]
@marcelomd Yeah, that would be useful. Would you like to contribute a patch? :)

**Comment by marcelomd** [comment]
Sure. I'll try to hack something... just have to learn perl first =)

**Comment by agentzh** [comment]
@marcelomd Cool. Thanks!
